lib/services.js

```diff
@@ -218,6 +218,19 @@ export default {
             authInfo.getAccountDisplayName();
         // This should be object creator's canonical ID.
         omVal['owner-id'] = authInfo.getCanonicalID();
+        // TODO: Find a way to store these headers for multipart uploads.
+        // params.headers is undefined when called from
+        // completeMultipartUploads because the latter does not pass
+        // headers as part of the params object
```

Comment:
I revised so that if 'aws-chunked' is sent as part of the Content-Encoding header it's removed. Let me know if it looks OK. To make sure, we don't use this value for our own logic in S3, correct?